### PR TITLE
Use @group to run only functional or unit tests

### PR DESCRIPTION
**Pull request by benja-M-1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,5 +8,5 @@ before_script:
   - php composer.phar --dev install
 
 script:
-  - phpunit -c phpunit.unit.xml.dist --coverage-text
-  - phpunit -c phpunit.functional.xml.dist --coverage-text
+  - phpunit -c phpunit.xml.dist --coverage-text --exclude-group functional
+  - phpunit -c phpunit.functional.xml.dist --coverage-text --group functional
```
Comment (adrienbrault) [inline .travis.yml]: Shouldn't it be phpunit.xml.dist instead of phpunit.functional.xml.dist ?

Comment (benja-M-1) [inline .travis.yml]: Yes it should :)

**Comment by adrienbrault** [comment]
Thanks!

File: Tests/Functional/ControllerTest.php [hunk 2]
```diff
@@ -24,6 +27,9 @@ public function testGetPostXml()
         $response->getContent());
     }
 
+    /**
+     * @group functional
+     */
```
Comment (stof) [inline Tests/Functional/ControllerTest.php:32]: you could move it on class level

**Pull request merged**